### PR TITLE
feat(cmd): add more information to shutdown log

### PR DESCRIPTION
**Pull request by PragmaTwice**
"Bye bye" seems useless and doesn't contain any information.

### CODE REVIEW

File: src/cli/main.cc [hunk 1]
```diff
@@ -47,7 +47,7 @@ Server *srv = nullptr;
 
 extern "C" void SignalHandler([[maybe_unused]] int sig) {
   if (srv && !srv->IsStopped()) {
-    LOG(INFO) << "Bye Bye";
+    LOG(INFO) << "Signal " << sig << " received, stoping the server";
```
Comment (PragmaTwice) [inline src/cli/main.cc]:
```suggestion
    LOG(INFO) << "Signal " << sig << " received, stopping the server";
```

File: src/commands/cmd_server.cc [hunk 2]
```diff
@@ -544,7 +544,7 @@ class CommandShutdown : public Commander {
     }
 
     if (!srv->IsStopped()) {
-      LOG(INFO) << "bye bye";
+      LOG(INFO) << "SHUTDOWN command received, stoping the server";
```
Comment (PragmaTwice) [inline src/commands/cmd_server.cc]:
```suggestion
      LOG(INFO) << "SHUTDOWN command received, stopping the server";
```

**Review by torwig — APPROVED** [review]
LGTM ()

**Review by git-hulk — APPROVED** [review]

File: src/cli/main.cc [hunk 3]
```diff
@@ -116,6 +116,7 @@ static void InitGoogleLog(const Config *config) {
 
 int main(int argc, char *argv[]) {
   srand(static_cast<unsigned>(util::GetTimeStamp()));
+  crc64_init();
```
Comment (mapleFU) [inline src/cli/main.cc:119]: Curious why crc64 change to here?

Comment (PragmaTwice) [reply src/cli/main.cc:119]: I think it should be in the beginning.

**Review by mapleFU — APPROVED** [review]

**Pull request merged**